lib/data/2024_hud_lists.json

```diff
@@ -1024,7 +1020,7 @@
     ]
   },
   {
-    "name": "LivingSituation",
+    "name": "CurrentLivingSituation",
```

Comment:
For 2024, switching to more explicit naming. This file contains lists for `CurrentLivingSituation`, `PriorLivingSituation`, and `Destinations`. IMO this makes the code a lot more readable.

Reply:
I like that.  There will be some duplication, but since it's code gen'd that seems fine, right?

Reply:
Yeah